.github/workflows/tests.yml

```diff
@@ -50,13 +52,13 @@ jobs:
       # https://github.com/codecov/codecov-action
       - name: Upload coverage results to Codecov
         uses: codecov/codecov-action@v1
-        if: always() && (matrix.php-version == '7.4' || matrix.php-version == '8.0')
+        if: always() && (matrix.php-version == '7.4')
```

Comment:
Can you bring this back and also adjust `.scrutinizer.yml` to say `runs: 2` again?

Reply:
code coverage is back needed to use phpunit 9 for it as 8 didn't support code coverage for php 8